.github/workflows/benchmarks.yml

```diff
@@ -85,7 +85,7 @@ jobs:
 
       - name: Update Baseline
         if: |
-          ((github.event_name == 'push' && github.ref == 'master') ||
+          ((github.event_name != 'pull_request' && github.ref == 'master') ||
```

Comment:
Will this just update the baseline every day with the nightly? Is that the behavior we want or do we still only want to update the baseline on new commits?

Reply:
yeah it would update the baseline every night. I think this is better since we'd get a better baseline from the additional data, compared to what we have now where the benchmarks are run infrequently - @erthalion do I understand this correctly?

Reply:
I spoke to @erthalion on Slack and he has agreed that it is a good opportunity to run update the baseline